seed/src/lib.rs

```diff
@@ -135,15 +137,15 @@ impl Default for NodeConfig {
 /// Seed node instance.
 pub struct Node {
     /// Sender end of user requests.
-    handle: chan::UnboundedSender<Request>,
+    handle: mpsc::Sender<Request>,
     /// Receiver end of user requests.
-    requests: chan::UnboundedReceiver<Request>,
+    requests: mpsc::Receiver<Request>,
 }
 
 impl Node {
     /// Create a new seed node.
     pub fn new() -> Result<Self, Error> {
-        let (handle, requests) = chan::unbounded::<Request>();
+        let (handle, requests) = mpsc::channel::<Request>(64);
```

Comment:
Should we parameterise this number?

Reply:
Maybe, what is the knob called? How is it exposed and contextualise for an operator? What would be a sanerisher default than 64?

Reply:
It represents the number of tasks that the `Node` can handle right? So it's kind of like a backpressure knob or something along those lines. 

> How is it exposed and contextualise for an operator?

How did you contextualise the need for it? :)

> What would be a sanerisher default than 64?

I would think that it would be some function of:
```
#expected_requests_per_second / #average_seconds_to_complete_request
```

Reply:
> How did you contextualise the need for it? :)

It's to address an unbounded queue, as contextualised in the commit message we've seen requests just pilling up because of polling nature of the UI when the tincans and machinery in the background was stuck, to then just explode with a release of all resources. It's one of the efforts to make the seed operation more resilient.

> I would think that it would be some function of:

I don't think either of those numbers are known and/or depend heavily on the local setup. While I think it's right to have a knob, it should mature more before we know what the knob does exactly and how it can be tuned. The surface of the seed is already expanding and I'd like to avoid a scenario where you need experts to tune the params right (JVM, aehm).

Reply:
> I don't think either of those numbers are known and/or depend heavily on the local setup. While I think it's right to have a knob, it should mature more before we know what the knob does exactly and how it can be tuned.

How can it mature without being easily tuned from the outside? The only way to change the value now would be to change this line of code, re-compile, and start the executable again. But obviously, we can start off with the "sensible" default of 64.

I don't feel too strongly about this though :)